site/_posts/2020-06-05-private-labeling-with-fusionauth.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog-post
+title: How To Use FusionAuth's Multi-Tenant Feature To Create A Private Label Offering
+description: You can leverage multi-tenant features to easily separate users, themes, API keys and more. This will allow for logical separation while still allowing your customer support and operations teams to manage one FusionAuth instance.
+author: Dan Moore
+image: blogs/private-label-with-tenants/how-to-use-fusionauths-multi-tenant-feature-to-create-a-private-label-offering.png
+category: blog
+tags: api-tenants
+excerpt_separator: "<!--more-->"
+---
+
+Sometimes you just need a little space, right? Tenants in FusionAuth can provide logical separation of users and applications while letting admins manage one FusionAuth installation.
+
+<!--more-->
+
+All editions of FusionAuth support multiple tenants within the same FusionAuth installation. Using this feature eases operational burdens while still maintaining logical divisions.
+
+## Why set up multi-tenant FusionAuth?
+
+Pretend you have a SaaS application which lets people manage their todos. It's a glorious task management application. Your company, TodoInc, sells accounts to individuals. That application lives at app.todo.com. You are using FusionAuth as your user data store, and you have other applications using it as well: a [forum application](/blog/2020/05/13/setting-up-single-sign-on-for-nodebb), [Zendesk](/docs/v1/tech/samlv2/zendesk) for customer support and [GSuite](/docs/v1/tech/samlv2/google). 
+
+However, Company1, a large multinational corporation with a big checkbook, approaches you. They want a premium edition; they have decided that your todo application is too good to live without. They want their application to be separate, private labeled, and located at todo.company1.com. They're willing to pay a premium price, as well.
+
+You realize you can offer this easily by modifying your application to respond to multiple hostnames. The logic is pretty straightforward.
+
+But what about your users? The accounts at todo.company1.com should be entirely separate from the accounts at app.todo.com. What if the CEO of Company1 has an account at app.todo.com and signs up for a corporate account? It doesn't make sense to mix personal and business tasks. Further, suppose you sell Organization2 a premium subscription as well. These users should be separate too. 
```

Comment:
Should we add in email addresses here to show that his/her account at `app.todo.com` is `alex@company1.com` and his/her account at `todo.company1.com` is also `alex@company1.com`? This is one of the main use-cases for Tenants, when a user needs two accounts that are logically separated but use the same `loginId`.